docs/quick-start.md

```diff
@@ -18,11 +18,11 @@ The Kubewarden stack comprises:
 
 * Some `ClusterAdmissionPolicy` resources: this is how policies are defined for Kubernetes clusters
 
-* Some `PolicyServer` resources: representing a deployment of a Kubewarden `PolicyServer`. Your administrator's policies are loaded and evaluated by the Kubewarden `PolicyServer`
+* Some `[[< policy-server-component >]]` resources: representing a deployment of a Kubewarden `[[< policy-server-component >]]`. Your administrator's policies are loaded and evaluated by the Kubewarden `[[< policy-server-component >]]`
```

Comment:
```suggestion
* Some `[[< policy-server-component >]]` resources: representing a deployment of a Kubewarden policy-server. Your administrator's policies are loaded and evaluated by the Kubewarden `[[< policy-server-component >]]`
```

Maybe?

As in, the Custom Resource PolicyServer represents deployment of the policy-server image/binary.

Reply:
I guess this is where we will, in due course, have [[< policy-server-deployment >]] mapping to policy-server.